source/renderer/app/components/wallet/send-form/WalletSendAssetsConfirmationDialog.js

```diff
@@ -434,14 +434,14 @@ export default class WalletSendAssetsConfirmationDialog extends Component<
                         assetIndex
                       );
                       return (
-                        <Fragment key={asset.fingerprint}>
+                        <Fragment key={asset.uniqueId}>
```

Comment:
I think this `Fragment` could be removed and the `<div>` below used as the single container element?

Reply:
There are 2 divs there, so it needs a Fragment (or `<>`, another `div` or any container).